client/src/app/site/motions/models/view-category.ts

```diff
@@ -0,0 +1,124 @@
+import { Category } from '../../../shared/models/motions/category';
+import { TranslateService } from '@ngx-translate/core';
+import { BaseModel } from '../../../shared/models/base/base-model';
+import { BaseViewModel } from '../../base/base-view-model';
+
+/**
+ * Category class for the View
+ *
+ * Stores a Category including all (implicit) references
+ * Provides "safe" access to variables and functions in {@link Category}
+ * @ignore
+ */
+export class ViewCategory extends BaseViewModel {
+    private _category: Category;
+    private _id: number;
+    private _name: string;
+    private _prefix: string;
+    private _edit: boolean;
+    private _synced: boolean;
+    private _opened: boolean;
+
+    public get category(): Category {
+        return this._category;
+    }
+
+    public get id(): number {
+        if (this.category) {
+            return this.category.id;
+        } else {
+            return null;
+        }
+    }
+
+    public get name(): string {
+        if (this.category) {
+            return this.category.name;
+        } else {
+            return null;
+        }
+    }
+
+    public get prefix(): string {
+        if (this.category) {
+            return this.category.prefix;
+        } else {
+            return null;
+        }
+    }
+
+    public set synced(bol: boolean) {
+        this._synced = bol;
+    }
+
+    public set edit(bol: boolean) {
+        this._edit = bol;
+    }
+
+    public set opened(bol: boolean) {
+        this._opened = bol;
+    }
+
+    public set prefix(pref: string) {
+        this._prefix = pref;
+        this._category.prefix = pref;
+    }
+
+    public set name(nam: string) {
```

Comment:
should be avoided. Name has to be the same like `_category.name`
Same goes for prefix and id.